CHANGELOG.md

```diff
@@ -17,14 +17,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  -  Added ability to delete sensors
  -  Alert for deploying and retrieving sensors when no event selected
  -  Added ability to create a peak
+ -  Added tooltips to all add button icons
  
 ### Changed
 
  -  Clear filters button styling to link
  -  Increased details modal widths
  -  Improved responsiveness of tables in details modals
  -  Autopopulate source and agency when adding file in site edit form
-
+ -  Disabled add button when no event selected for Peaks, Sensors, HWMs, Ref Datums and files (except site files)

```

Comment:
```suggestion
 -  Disabled add button when no event selected for Peaks, Sensors, HWMs and files (except site files)

```

You did good and did not implement for ref datums despite me including that in the issue title. The should in fact be able to create those as they are not event specific